FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -167,7 +167,7 @@ await foreach (FhirToDataLakeProcessingJobInputData input in inputs.WithCancella
                 _metricsLogger.LogResourceLatencyMetric(jobLatency.TotalSeconds);
 
                 _diagnosticLogger.LogInformation($"Finish FhirToDataLake job. Synced result data to {_inputData.DataEndTime}.");
```

Comment:
Do we also need to update diagnosticLogger?

Reply:
I think we don't need to update the diagnosticLogger as the customer should not know the job detail.